### ISSUE TITLE
Low performance & Question about some details in Table2

### ISSUE THREAD

**Issue by Senwang98** [duzhixing/FRS#2]
Hi, @duzhixing 
Thanks for your work.
In Table 2. of your paper,  Retina-Res50(student) has two different mAP performance (37.4 vs. 36.2).
I wonder Retina_R50@mAP=36.2 means??
Wishing for your reply, thanks!

**Comment by duzhixing** [comment]
In the lower part of Table 2 , Retian-Resnet50 is based on the mmdetection 1.2.
You can see the footnote：
For fair comparison, we run the experiment using the teacher and student model of the same accuracy, the
implementation is based on the mmdetection 1.2. In addition to this set of experiments, our other experiments
are based on mmdet2

**Comment by Senwang98** [comment]
@duzhixing 
Okay, got it!
by the way, is it convenient for you to share a training log of the RetinaNet-R50？(I try to run retinanet with teacher-res101-2x, so I want to know the if my training loss is right. No matter random seed or any log about retinanet will be ok!)

**Comment by Senwang98** [comment]
@duzhixing 
Hi, I have already run your code twice. But the default config you provide of retinanet50 has low performance.
![image](https://user-images.githubusercontent.com/23122139/146662043-4d83061e-ff39-41d6-979a-9e7ccb644e4c.png)
Here is my training log:
https://drive.google.com/file/d/1eZCXCTz7f0drcYRG-d1JWdvz0Fy0uLwD/view?usp=sharing
My config:
https://drive.google.com/file/d/1MjjKLrz1KM2OeXzWz-QZ5qc5pGLNSyhe/view?usp=sharing
------------------------------
I still run the faster-rcnn with total epoch =12. Student performance is expect to be 37.4+, but the finnal performance is only 36.6. Can you help me solve this peoblem?
![image](https://user-images.githubusercontent.com/23122139/146724979-37ddab4a-47da-40d6-8da1-0f388088fb97.png)
Faster-RCNN training log:
https://drive.google.com/file/d/1FHyVRtdGP85G82plW8tvCu6g7LG7CjOr/view?usp=sharing
corresponding config file:
https://drive.google.com/file/d/1QVeK5agTNdfAly_hP9tTpWSNnFLIsm2u/view?usp=sharing

**Comment by Senwang98** [comment]
Solve this problem by using mmdet=2.11.0 instaed of mmdet>2.13.0

**Comment by WayneMao** [comment]
Hi, @Senwang98 
Thanks for your advice.
I encountered the same situation. I ran RetinaNet 1x , the finnal performance is only 36.9. And my version of mmdet is 2.19.1.   
I‘ll try mmdet=2.11.0
![image](https://user-images.githubusercontent.com/35285052/147030566-f6929967-f255-4aad-95f4-9f23687741c0.png)

**Comment by Senwang98** [comment]
@WayneMao 
trust me, I have already used mmdet2.11 to meet the paper's mAP.
The author's code is ok!

**Comment by Lhj-cd** [comment]
> @WayneMao trust me, I have already used mmdet2.11 to meet the paper's mAP. The author's code is ok!

Hi, thank you for the large amount of useful information !!  @Senwang98 
You said you reproduced the points of the paper, could you please share the rereproduced training log? (Faster-rcnn和RetinaNet if convenient). I want to look at the number of distillation losses.
Thank you very much!

**Comment by Senwang98** [comment]
@HaojieIu 
I am sorry, detection-kd releated work was done when I am an intern. But now I have checked out!
All training logs were not available to me. 
I think this repo is easy to reproduct, I remember my mAP is only 0.1 lower than that in original paper.
So, if your mAP is much lower than author's, maybe you should check your codes carefully.
Hope my answer is helpful.

**Comment by Lhj-cd** [comment]
@Senwang98 
Thank you for your reply !!
I didn't reproduce the author's code, I wanted to apply the author's method to other fields besides detection, and wanted to know the approximate ratio of distillation loss to total loss. But it didn't seem to work.
